src/modules/UI/Wallets/action.js

```diff
@@ -436,6 +438,30 @@ export function addNewTokenThenDeleteOldSuccess (data: any) {
   }
 }
 
+export const updateWalletLoadingProgress = (walletId: string, newWalletProgress: number) => (dispatch: Dispatch, getState: GetState) => {
+  const data = {
+    walletId,
+    addressLoadingProgress: newWalletProgress
+  }
+  const state = getState()
+  const currentWalletProgress = state.ui.wallets.walletLoadingProgress[walletId]
+  const marginalProgress = newWalletProgress - currentWalletProgress
+
+  if (newWalletProgress !== 1 && marginalProgress < 0.1) return
+
+  dispatch({
+    type: UPDATE_WALLET_LOADING_PROGRESS,
+    data
+  })
+}
+
+export function insertWalletIdsForProgress (activeWalletIds: Array<string>) {
```

Comment:
Please make the value of `action.data` and object
```return {
  type: '...',
  data: { activeWalletIds }
}
```

Reply:
Done